contrib/authors_update.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -eu
+
+range="${1?Provide commit range like orig/v1.17..orig/v1.18}"
+
+tmp=AUTHORS.email
+grep @ AUTHORS >$tmp
+
+git log --no-merges --pretty=format:"%an%x09%ae" "$range" | sort | uniq | \
```

Comment:
maybe check if `git log` produces any output? add a log?

Reply:
fixed

Reply:
maybe add a log if it doesn't produces any output?

Reply:
fixed, but I only see something like "HEAD..HEAD" to do that..